docs/en_US/guides/faq.md

```diff
@@ -111,15 +111,17 @@ While this can be easily figured out by checking <router-link to="/get-started">
 - **A7 iPads**: iOS 12.5.7 (via Odysseyra1n or Chimera)
 - **A7/A8 iPhones**: iOS 12.5.7 (via Odysseyra1n or Chimera)
 - **iPod Touch (6th Generation)**: iOS 12.5.7 (via Odysseyra1n or Chimera)
-- **A8(X) iPads**: iPadOS 15.8.2 (via Dopamine, meowbrek2, or palera1n)
-- **iPod Touch (7th Generation)**: iOS 15.8.2 (via Dopamine, meowbrek2, or palera1n)
-- **A9/A10 iPhones**: iOS 15.8.2 (via Dopamine, meowbrek2, or palera1n)
-- **A9(X) iPads**: iPadOS 16.7.8 (via palera1n)
-- **A10(X) iPads**: iPadOS 17.5 (via palera1n)
-- **A11 iPhones**: iOS 16.7.8 (via palera1n)
+- **A8/A10X Apple TVs**: tvOS 17.6 (via palera1n)
+- **A8(X) iPads**: iPadOS 15.8.3 (via Dopamine, meowbrek2, or palera1n)
+- **iPod Touch (7th Generation)**: iOS 15.8.3 (via Dopamine, meowbrek2, or palera1n)
+- **A9/A10 iPhones**: iOS 15.8.3 (via Dopamine, meowbrek2, or palera1n)
+- **A9(X) iPads**: iPadOS 16.7.10 (via palera1n)
+- **A10(X) iPads**: iPadOS 17.6.1 (via palera1n)
+- **A11 iPhones**: iOS 16.7.10 (via palera1n)
 - **A12-A14 iPhones**: iOS 16.5.1 (via Dopamine)
 - **A12-A14/M1 iPads**: iPadOS 16.5.1 (via Dopamine)
 - **A15/A16 iPhones**: iOS 16.5 (via Dopamine)
 - **A15/M2 iPads**: iPadOS 16.5 (via Dopamine)
+- **T2 (Intel Macs)**: bridgeOS 8.6 (macOS 12.7.6/13.6.9/14.6.1) (via palera1n)
```

Comment:
As the guide doesn't cover T2 Macs, this part should be removed.

Also make sure that the latestfw file is bumped to 17.6.1, but otherwise, this looks fine

Reply:
Ripped that line out, latestfw also bumped